src/components/MaximizedLogoIcon/MaximizedLogoIcon.tsx

```diff
@@ -0,0 +1,40 @@
+      fill={color}
+    />
+    <path
+      fillRule="evenodd"
+      clipRule="evenodd"
+      d="M65.9363 24.5938V20.0438L68.9925 24.5938H72.6679L69.432 20.0641C71.2097 19.6172 72.2684 18.4188 72.2684 16.286C72.2684 13.9906 70.5905 12.4063 68.0137 12.4063H63V24.5938H65.9363ZM68.2934 17.6875H65.9363V14.9453H68.2135C68.8727 14.9453 69.392 15.575 69.392 16.286C69.392 16.9969 68.9526 17.6875 68.2934 17.6875ZM82.1418 24.5938V22.0344H76.9283V19.8406H81.5425V17.2406H76.9283V15.0469H81.9221V12.4063H73.992V24.5938H82.1418ZM86.3622 24.5938L86.9415 23.3344H91.6756L92.2748 24.5938H95.4309L89.5582 12H89.0389L83.2062 24.5938H86.3622ZM90.6768 21.2016H87.8604L89.2586 18.1141H89.2986L90.6768 21.2016ZM101.809 25C104.026 25 105.684 24.0453 106.882 22.7656L104.925 20.8156C104.086 21.6485 103.007 22.1766 101.809 22.1766C99.6713 22.1766 98.293 20.5516 98.293 18.5C98.293 16.4485 99.6713 14.8235 101.809 14.8235C103.007 14.8235 104.086 15.3516 104.925 16.1844L106.942 14.2344C105.664 12.8735 104.086 12 101.809 12C98.0134 12 95.3167 14.8641 95.3167 18.5C95.3167 22.136 98.0134 25 101.809 25ZM113.719 24.5938V15.0469H116.436V12.4063H108.087V15.0469H110.763V24.5938H113.719ZM121.016 24.5938V12.4063H118.08V24.5938H121.016ZM128.113 25L133.406 12.4063H130.35L127.933 18.2766L125.516 12.4063H122.46L127.753 25H128.113ZM143 24.5938V22.0344H137.787V19.8406H142.401V17.2406H137.787V15.0469H142.78V12.4063H134.85V24.5938H143Z"
+      fill={color}
```

Comment:
Is it possible to compose the logo text SVG paths with LogoIcon path used in the 'LogoIcon' component instead? The paths seem slightly different, but they might not have to be as the two logos look pretty much the exact same. Perhaps there could be two logo components, LogoText and LogoIcon that can be composed to make a MaximizedLogo.